### PR TITLE
Added token login feature. Still need to make token validator

### PR DESCRIPTION
**Pull request by TheKriserOne**

### CODE REVIEW

File: loginInfo
Comment (laycookie) [inline loginInfo:1]: Got to send this to `.gitignore`

Also this should probably be in a public folder or something

**Review by laycookie — CHANGES_REQUESTED** [review]
Login info file should be gitignored and should be located in a public folder

File: loginInfo
Comment (TheKriserOne) [reply loginInfo:1]: doesnt work with the file path you specified, let me fix it

**Review by laycookie — COMMENTED** [review]
Looks fixed now

**Review by laycookie — APPROVED** [review]

**Pull request merged**